plugins/users/users_inspector.rb

```
@@ -25,8 +25,10 @@ def initialize(system, description)
 
   def inspect(_filter, _options = {})
     passwd = @system.read_file("/etc/passwd")
-    shadow = @system.read_file("/etc/shadow")
-
+    begin
+      shadow = @system.read_file("/etc/shadow")
+    rescue => e
```

Don't silently ignore exceptions.